README.rst

```diff
@@ -36,26 +36,30 @@ Incomplete support for creating slave nodes also exists.
 Installation
 ------------
 
-Install from PyPI_ using pip::
+Install from PyPI_ using :program:`pip`::
 
-    $ pip install canopen
+    $ python3 -m pip install canopen
 
-Install from latest master on GitHub::
+Install from latest ``master`` on GitHub::
 
-    $ pip install https://github.com/christiansandberg/canopen/archive/master.zip
+    $ python3 -m pip install https://github.com/christiansandberg/canopen/archive/master.zip
 
 If you want to be able to change the code while using it, clone it then install
 it in `develop mode`_::
 
     $ git clone https://github.com/christiansandberg/canopen.git
     $ cd canopen
-    $ pip install -e .
+    $ python3 -m pip install -e .
 
 Unit tests can be run using the pytest_ framework::
 
-    $ pip install pytest
+    $ python3 -m pip install pytest
     $ pytest -v
 
+You can also use :mod:`unittest` standard library module::
+
+    $ python3 -m unittest discover test -v
```

Comment:
I dont think we should rely on unittest anymore. It's fine to write new tests using pytest.

Reply:
I'm not sure how to interpret this. Do you mean that you don't want to promote running tests using `unittest`, or do you propose to do a blanket change and rewrite all tests using `pytest` APIs?

Reply:
I understand it as better keep recommending `pytest` only and for *new* tests, target that API.  If someone has the time to port over the existing tests, go ahead.  I don't have a strong opinion here, as long as the tests can be invoked easily.

Reply:
What's the rationale? `pytest` understands `unittest` perfectly well. If you want to run the test suite using `pytest`, go ahead. Ditto for `unittest`. Using `unittest` to write the tests makes for greater freedom when running the test suite. Anyway, this is better discussed in a separate issue.

Reply:
:man_shrugging:

Let's not hold this up by such a minor detail.  If anyone has strong feelings about it, let's handle in a new issue.